server.js

```diff
@@ -25,7 +25,6 @@ const CFG_PATH = path.join(PATH, 'config');
   for (let id = 0; id < count; id++) start(id);
 
   const stop = async () => {
-    console.log();
```

Comment:
We need this for pretty terminal output: new line after `^C` and before graceful shutdown messages. But we may switch `stdin` to raw-mode (this will prevent printing `^C`). I'll do it later.

Reply:
Okay, good.